depends/packages/libevent.mk

```diff
@@ -3,27 +3,30 @@ $(package)_version=2.1.12-stable
 $(package)_download_path=https://github.com/libevent/libevent/releases/download/release-$($(package)_version)/
 $(package)_file_name=$(package)-$($(package)_version).tar.gz
 $(package)_sha256_hash=92e6de1be9ec176428fd2367677e61ceffc2ee1cb119035037a27d346b0403bb
+$(package)_patches=cmake_fixups.patch
+$(package)_build_subdir=build
 
 # When building for Windows, we set _WIN32_WINNT to target the same Windows
 # version as we do in configure. Due to quirks in libevents build system, this
 # is also required to enable support for ipv6. See #19375.
 define $(package)_set_vars
-  $(package)_config_opts=--disable-shared --disable-openssl --disable-libevent-regress --disable-samples
-  $(package)_config_opts += --disable-dependency-tracking --enable-option-checking
-  $(package)_config_opts_release=--disable-debug-mode
+  $(package)_config_opts=-DEVENT__DISABLE_BENCHMARK=ON -DEVENT__DISABLE_OPENSSL=ON
+  $(package)_config_opts+=-DEVENT__DISABLE_SAMPLES=ON -DEVENT__DISABLE_REGRESS=ON
+  $(package)_config_opts+=-DEVENT__DISABLE_TESTS=ON -DEVENT__LIBRARY_TYPE=STATIC
   $(package)_cppflags_mingw32=-D_WIN32_WINNT=0x0601
+  $(package)_config_opts_debug+=-DEVENT__DISABLE_DEBUG_MODE=ON
```

Comment:
Could this just always be disabled?

Reply:
Probably. Happy to just drop this, and it could always be added later.

Reply:
That's fine too, it is not used anyway.